### DISCUSSION TITLE
[GPSR] Dane producenta bezpośrednio podczas tworzenia oferty bez zapisywania na koncie.

### DISCUSSION THREAD

**Discussion by creativepl-php** [allegro/allegro-api#10552] [Suggestions - rejected]
Proponuję, aby w polach np. productSet podać od razu dane producenta, bez konieczności tworzenia go. Podany producent byłby od razu wyświetlany w ofercie. Aktualnie podaje się id lub nazwę producenta - moim zdaniem, skoro to na sprzedawcy spoczywa odpowiedzialność, to sprzedawca powinien mieć elastyczność w tym, czy chce jednorazowo przekazać dane producenta do oferty, czy zapisać go sobie na Allegro i użyć później. 

Dla Allegro to żadna różnica czy producent jest zapisany czy nie, bo przecież nie ma żadnego zasobu do pobrania producenta z "bazy producentów" (takiej bazy nie ma). Przede wszystkim pomoże to tworzyć taką bazę producentów w aplikacjach. Odchodziłby też problem aktualizacji danych, bo te najnowsze można by było od razu w parametrze podać. Uniknęłoby to też dublowania danych. Bez zbędnych requestów.

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za sugestię. Na tę chwilę nie mamy takich planów, jednak jeśli zdecydujemy się na wprowadzenie takiego rozwiązania, poinformujemy w wątku.

**Comment by MaciejFrackowiak** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany. Obecnie umożliwiamy [wystawienie](https://developer.allegro.pl/news/gpsr-uzupelnilismy-katalog-produktow-o-informacje-o-bezpieczenstwie-produktu-q0PgaG8EoFd)/[edycję](https://developer.allegro.pl/news/zarzadzanie-ofertami-usprawnilismy-aktualizacje-informacji-o-gpsr-6MO84lwMmTe) oferty z uwzględnieniem danych z Katalogu Produtków Allegro wzbogaconych o informacje o bezpieczeństwie produktu.
